manage-data/data-store/data-streams.md

```diff
@@ -13,7 +13,7 @@ products:
 
 # Data streams [data-streams]
 
-A data stream lets you store append-only time series data across multiple indices while giving you a single named resource for requests. Data streams are well-suited for logs, events, metrics, and other continuously generated data.
+A data stream represents an index abstraction that's optimized for storing append-only time series data. It stores data across multiple backing indices while giving you a single named resource to use for requests. Data streams are well-suited for logs, events, metrics, and other continuously generated data.
```

Comment:
```suggestion
A data stream represents an index abstraction that's optimized for storing append-only time series data. It stores data across multiple backing indices while giving you a single named resource to use for requests. Data streams are well-suited for logs, events, metrics, and other continuously generated data.
```

I wonder if we could improve on "represents an index abstraction". I'm not sure that I'd gather what that means as someone new to data streams.

What would you think of either of these?

`A data stream acts as a layer of abstraction over a set of indices that are optimized for storing append-only time series data.`

`A data stream acts as a single named resource for a set of indices that are optimized for storing append-only time series data.`